src/Helper/Layout.php

```diff
@@ -100,11 +100,9 @@ protected function getViewModelHelper()
     {
         if (! $this->viewModelHelper) {
             /**
-             * @psalm-suppress DeprecatedMethod
+             * @psalm-suppress PossiblyNullReference, UndefinedInterfaceMethod
```

Comment:
I think we shouldn't suppress this, but rather baseline it: the problem is there, but we just didn't fix it yet.

Reply:
Yeah was just trying to keep the CI chain happy.  You're ok with a psalm break?

Reply:
Nono, just regenerate the baseline file 👍